service-api/src/main/java/greencity/client/UserRemoteClient.java

```diff
@@ -106,11 +107,19 @@ public interface UserRemoteClient {
     void signUpEmployee(@RequestBody EmployeeSignUpDto dto);
 
     /**
-     * Edit an employee's email.
+     * Update employee email.
      *
-     * @param employeeEmail    {@link String}
-     * @param newEmployeeEmail {@link String}
+     * @param newEmployeeEmail - new email of employee.
+     * @param uuid             - uuid of current employee.
      */
-    @PutMapping("user/update-employee-email")
-    void updateEmployeeEmail(@RequestParam String employeeEmail, @RequestParam String newEmployeeEmail);
+    @PutMapping("/user/employee-email")
+    void updateEmployeeEmail(@RequestParam String newEmployeeEmail, @RequestParam String uuid);
+
+    /**
+     * Update authorities in chosen employee.
+     *
+     * @param dto contains email and list of positions.
```

Comment:
please add a link to the `UpdateEmployeeAuthoritiesDto`

Reply:
added